rosdep/base.yaml

```diff
@@ -4994,6 +4994,11 @@ r-base-dev:
   fedora: [R-devel]
   gentoo: [dev-lang/R]
   ubuntu: [r-base-dev]
+range-v3:
+  debian: [librange-v3-dev]
+  fedora: [range-v3]
+  gentoo: [dev-cpp/range-v3]
+  ubuntu: [librange-v3-dev]
```

Comment:
Same comment for ubuntu; this package is only available on eoan and later, but we still support xenial and bionic for Kinetic and Melodic, respectively.  This key will need to be more specific.

Reply:
It's available for [bionic](https://packages.ubuntu.com/bionic/librange-v3-dev) as well.